kong-1.0.3-0.rockspec

```diff
@@ -128,6 +129,7 @@ build = {
     ["kong.db.dao.snis"] = "kong/db/dao/snis.lua",
     ["kong.db.dao.targets"] = "kong/db/dao/targets.lua",
     ["kong.db.dao.plugins"] = "kong/db/dao/plugins.lua",
+    ["kong.db.declarative"] = "kong/db/declarative/init.lua",
```

Comment:
this also should not be in this commit, but rather in the commit where the module is introduced